### PR TITLE
fix(hydra): don't running any python code outside hydra image

### PR DESCRIPTION
**Pull request by fruch**
when doing so we can easily break users depends on which python version they are using, we shouldn't assume anything about user env.

### Testing

- [ ]

### PR pre-checks (self review)

- [x] I added the relevant `backport` labels
- [x] I didn't leave commented-out/debugging code

### Reminders

- Add New configuration option and document them (in `sdcm/sct_config.py`)
- Add unit tests to cover my changes (under `unit-test/` folder)
- Update the Readme/doc folder relevant to this change (if needed)

### CODE REVIEW

File: docker/env/hydra.sh [hunk 1]
```diff
@@ -245,8 +244,6 @@ function run_in_docker () {
     $([[ -n "$HYDRA_DRY_RUN" ]] && echo echo) \
     $tool ${REMOTE_DOCKER_HOST} run --rm ${TTY_STDIN} --privileged \
         -h ${HOST_NAME} \
-        -l "TestId=${SCT_TEST_ID}" \
```
Comment (vponomaryov) [inline docker/env/hydra.sh:248]: Why removing SCT_TEST_ID?

File: docker/env/hydra.sh [hunk 2]
```diff
@@ -14,7 +14,6 @@ SCT_DIR=$(dirname "${DOCKER_ENV_DIR}")
 SCT_DIR=$(dirname "${SCT_DIR}")
 VERSION=v$(cat "${DOCKER_ENV_DIR}/version")
 HOST_NAME=SCT-CONTAINER
-RUN_BY_USER=$(python3 "${SCT_DIR}/sdcm/utils/get_username.py")
```
[the remark this answers is not in the document]
Comment (fruch) [reply docker/env/hydra.sh:17]: it's for labeling the hydra command itself, it's pointless

it from back when we have builder used from multiple tests at the same time, and we need to differentiated them from each other

now that we have  sct-runner it's doesn't help us much.

File: docker/env/hydra.sh (hunk 1 above)
Comment (fruch) [reply docker/env/hydra.sh:248]: same as below, it's meant for when we were running lots of test at the same time on the builders

we aren't doing that anymore, hence those labels are not really needed, or helpful.

Comment (fruch) [reply docker/env/hydra.sh:248]: Unittest fails cause both of them, and would need to amended, either way

**Review by soyacz — APPROVED** [review]
LGTM

**Review by vponomaryov — APPROVED** [review]
LGTM

**Review by roydahan — APPROVED** [review]

**Pull request merged**